src/styled.tsx

```diff
@@ -1,12 +1,15 @@
+import * as React from 'react';
 import styled from '@emotion/styled';
 import colors from './colors';
 
 export const paraGraphDefaultStyle = {
-  fontSize: 14,
-  lineHeight: '22px',
+    fontSize: 14,
+    lineHeight: '22px',
 };
 
-const Button = styled.button`
+const Button = styled((props: any) => {
+    return <button {...props} type="button" />
+})`
```

Comment:
I think you can use [`.attrs`](https://styled-components.com/docs/api).

```suggestion
const Button = styled.button.attrs(() => ({
  type: 'button',
})`
```

Reply:
Thanks for that suggestion!

Reply:
Thanks! Is it possible to check if it works locally?

Reply:
awesome suggestion from @kotarella1110 !

Reply:
Actually i tested this, and the suggestion is based on the styled-components API. Since we are using emotion, I had to try something else. I used a defaultProps on the Button type instead (was suggested in an issue in the emotion repo)

Reply:
https://github.com/emotion-js/emotion/issues/821#issuecomment-415697393

Reply:
oh... sorry! thanks!
you are awesome 👍

Reply:
Thanks! Your suggestion made me want to make it cleaner!